File: 코딩테스트 입문/Javascript/홀짝에 따라 다른 값 반환하기.js
function solution(n) {
    let answer = 0;
    
    let setNum = (n%2==0) ? 2 : 1;
    
    for(let i=setNum; i<=n; i+=2){
        answer += Math.pow(i,setNum);
    }
    
    return answer;
}

File: 코딩테스트 입문/Python/홀짝에 따라 다른 값 반환하기.py
def solution(n):
    answer = 0
    
    # 홀수인 경우
    if n%2!=0:
        for i in range(1,n+1,2):
            answer += i
    else:
        for i in range(2,n+1,2):
            answer += pow(i,2)
            
    return answer